Testscripts/Linux/XDPDumpSetup.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# This script installs XDP dump application
+
+# Helper Function
+# Install dependencies for XDP
+function Install_XDP_Dependencies(){
+
+    if [ -z "${1}" -o -z "${2}" ]; then
+        LogErr "ERROR: must provide install ip and distro to Install_XDP_Dependencies()"
+        SetTestStateAborted
+        exit 1
+    fi
+
+    local install_ip="${1}"
+    local distro="${2}"
```

Comment:
you get distro here, why invoke GetDistro again in the below line? Put GetDistro before invoke Install_XDP_Dependencies